### ISSUE TITLE
Is there no search?

### ISSUE THREAD

**Issue by hellonearthis** [WebAssembly/website#141]
The website has no search?  I could not find a search.

**Comment by binji** [comment]
No, but you can use site `site:webassembly.org` (or similar) in search engines for this.

**Comment by tomayac** [comment]
Closing as per the workaround in https://github.com/WebAssembly/website/issues/141#issuecomment-500779018.
